static-site/gatsby-node.js

```diff
@@ -193,6 +193,13 @@ exports.createPages = ({graphql, actions}) => {
           component: path.resolve("src/sections/users.jsx")
         });
 
+        // Create individual groups pages
+        createPage({
+          path: "/groups/:groupName",
+          matchPath: "/groups/:groupName",
```

Comment:
Is there any advantage of having this here, when it's the same as `path`? (Gatsby docs [here](https://www.gatsbyjs.com/docs/gatsby-internals-terminology/#matchpath)).

Reply:
I think it's vital; commenting it out like:
```
        createPage({
          path: "/groups/:groupName",
          // matchPath: "/groups/:groupName",
          component: path.resolve("src/sections/individual-group-page.jsx")
        });
```
, I get redirected to http://localhost:5000/ immediately upon visiting http://localhost:5000/groups/seattleflu

Reply:
Meanwhile I'm not sure `path` being exactly `/groups/:groupName` allows this to work in any way other than making a distinction from the `/groups` page for which we want to return a different page:
```
        createPage({
          path: "/groups",
          component: path.resolve("src/pages/groups.jsx")
        });
```